src/app/components/Grid/index.stories.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import Grid, { GridMaxWidthGroup4 } from '.';
+
+const outerGridProps = {
+  columns: {
+    group0: 6,
+    group1: 6,
+    group2: 6,
+    group3: 6,
+    group4: 6,
+    group5: 6,
+  },
+};
+
+const gridItemProps = {
+  item: true,
+  columns: {
+    group0: 2,
+    group1: 2,
+    group2: 2,
+    group3: 2,
+    group4: 2,
+    group5: 2,
+  },
+};
+
+storiesOf('Components|Grid', module)
+  .add('default', () => {
+    return (
+      <Grid {...outerGridProps}>
+        <Grid {...gridItemProps}>
+          <p>Test</p>
```

Comment:
Shall we put a useful example? Otherwise, you can not really tell what are the differences between the three stories.